source/Applications/Basic/FileFormats/ZDF2PLY/Properties/AssemblyInfo.cs

```diff
@@ -31,6 +32,7 @@
 //
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
```

Comment:
Agree with @apartridge , comment is weird now.

Reply:
// You can specify all the values or you can default the Build and Revision Numbers
// as shown below:

[assembly: AssemblyVersion("1.0")]
[assembly: AssemblyFileVersion("1.0")]
[assembly: AssemblyInformationalVersion("1.0")]

Will this be ok? @apartridge @torbsorb

Reply:
If so, you can review again. It is implemented.